src/integrationTest/java/uk/gov/hmcts/reform/demo/config/SwaggerPublisher.java

```diff
@@ -27,19 +24,12 @@
     @Autowired
     private MockMvc mvc;
 
-    @DisplayName("Generate swagger documentation")
+    @DisplayName("Generate swagger v2 documentation")
     @Test
     @SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
     void generateDocs() throws Exception {
-        byte[] specs = mvc.perform(get("/v2/api-docs"))
+        mvc.perform(get("/v2/api-docs"))
             .andExpect(status().isOk())
-            .andReturn()
-            .getResponse()
-            .getContentAsByteArray();
-
-        try (OutputStream outputStream = Files.newOutputStream(Paths.get("/tmp/swagger-specs.json"))) {
```

Comment:
This isn't really a test, it's firing up the app in a mock context to get the generated swagger spec,
its then published by travis to our swagger docs
https://github.com/hmcts/reform-api-docs/tree/master/docs/specs

https://hmcts.github.io/reform-api-docs/swagger.html?url=https://hmcts.github.io/reform-api-docs/specs/document-management-store-app.json

Reply:
but isn't it part of the travis file already?
https://github.com/hmcts/spring-boot-template/blob/master/.travis.yml

Reply:
that publishes the spec, this generates it